server/data_access_layer/daos/chart_report.dao.js

```diff
@@ -98,4 +98,23 @@ exports.deleteChartReportById = async(chartReportId, chartReportModel = ChartRep
                 reject(response);
             });
     });
+}
+
+// Reports Types related functions
+exports.getPerformanceReportsWhenConnectedAsAdmin = async (userId, performanceReportModel = PerformanceReportModel) => {
```

Comment:
I'm not sure why this function is in the Chart Report Dao. I think it should be in the report dao

Reply:
Was going to say the same thing

Reply:
Thinking about it, it makes more sense in the report dao, I'll change it now